scripts/setup/setup.bat

```diff
@@ -27,19 +27,26 @@ IF ERRORLEVEL 1 (
 where /q node
 IF ERRORLEVEL 1 (
 	ECHO Error: node is not installed.
-	ECHO Please Install NodeJS from: https://nodejs.org/en/download/
+	ECHO Please Install NodeJS from: https://nodejs.org/en/download
 	ECHO And make sure its in the path
     GOTO :end
 )
 
 where /q npm
 IF ERRORLEVEL 1 (
 	ECHO 'Error: npm is not installed.' >&2
-	ECHO Please install npm from: https://nodejs.org/en/download/
+	ECHO Please install npm from: https://nodejs.org/en/download
 	ECHO And make sure its in the path
     GOTO :end
 )
-echo Dependencies are already Installed
+
+where /q pnpm
+IF ERRORLEVEL 1 (
+	ECHO PNPM is not installed but he is optionnal

```

Comment:
same here, do not mention that pnpm can be installed optionally, instead just use it if its available

Reply:
I think this should be hard to code this but i will try (any help would be appreciated)

Reply:
also change the wording from he to it